bpfctl/src/main.rs

```diff
@@ -165,6 +165,31 @@ enum LoadCommands {
         #[clap(short, long, verbatim_doc_comment)]
         tracepoint: String,
     },
+    /// Install an eBPF uprobe
+    Uprobe {
+        /// Optional: function to attach the uprobe to.
+        #[clap(short, long)]
+        fn_name: Option<String>,
+
+        /// Optional: offset added to the address of the target function
+        /// (or beginning of target if no function is identified)
+        #[clap(short, long)]
+        offset: Option<u64>,
+
+        /// Required: absolute path to a binary or library, or a library name
+        /// Example: --target "libc".
+        #[clap(short, long)]
+        target: String,
+
+        /// Optional: process identification number (PID) to run uprobe for.
```

Comment:
```suggestion
        /// Optional: process identification number (PID) to attach uprobe to.
```

Reply:
I think that technically, the uprobe is attached to the function in the target, but if the PID is set, it only executes when that function is executed by the given PID.  The doc comment from the Aya code is below:
```rust
    /// Attaches the uprobe to the function `fn_name` defined in the `target`.
    /// If `offset` is non-zero, it is added to the address of the target
    /// function. If `pid` is not `None`, the program executes only when the target
    /// function is executed by the given `pid`.
```

Reply:
Changed to the following:
"only execute uprobe for given process identification number (PID)"